src/components/OpenSource/WhyOpenSource.js

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from 'react'
+import { Padding } from 'styled-components-spacing'
+
+import TitleAndListWithOptionalLogos from '../Common/TitleAndListWithOptionalLogos'
+import Subtitle from '../Typography/Subtitle'
+import LogoGrid from '../Common/LogoGrid'
+
+const WhyOpenSource = ({ title, list, subtitle, extraContent, companies }) => {
+  return (
+    <Padding>
```

Comment:
We're not using `Padding` component anymore

Use `styled-components` and create your own component using the `props.theme.spacing[n]`